docs/GetStarted/getting_started_android_cmake.md

```diff
@@ -77,7 +84,31 @@ $ cmake -G Ninja -B build-android/  \
     does [not support](https://github.com/google/iree/issues/1269) GCC well at
     the moment.
 
-Build all targets:
+### Configure on Windows
+
+On Windows, we will need the full path to the `cl.exe` compiler. This can be
+obtained by [opening a developer command prompt window](https://docs.microsoft.com/en-us/cpp/build/building-on-the-command-line?view=vs-2019#developer_command_prompt) and type
+`where cl.exe`. Copy the path and substitue all `\` with `/` to get the
+CMake-style path. Then in a command prompt (`cmd.exe`):
```

Comment:
Eh, this full path is annoying. I tried some variations on it but none of them worked :/

Reply:
Yup. Might be easier to define a local variable so we don't repeat ourselves multiple times. The dev environment on Windows is just so.. diverse. We have native cmd/powershell, Cygwin, MYSIS, ... LLVM is just quite confused when cross compiling with Android toolchain and *guessing* the host environment. So we need all the extra configurations here. And the logic in LLVM to figure out host triple is non-trivial; I wasn't able to spot where to fix easily. So this is it for now. :)